source/CMakeLists.txt

```diff
@@ -122,7 +122,11 @@ else()
     clangAST
     clangFrontend
     )
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${LLVM_COMPILE_FLAGS}")
+  list(REMOVE_ITEM CMAKE_CXX_FLAGS "-fno-exceptions")
+  string (REPLACE "-fno-exceptions" "" CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS})
+  message(STATUS "binder: CMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}")
 endif()
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/binder
+install(TARGETS binder
 PERMISSIONS WORLD_EXECUTE WORLD_READ OWNER_WRITE OWNER_READ OWNER_EXECUTE
-DESTINATION ${CMAKE_INSTALL_BINDIR})
+DESTINATION binder/bin COMPONENT binder)
```

Comment:
Please leave the ${CMAKE_INSTALL_BINDIR} as it is. The 'COMPONENT' is ok though.